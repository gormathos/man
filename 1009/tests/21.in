sol_id;user_id;task_id;score;posted_time
116412;1011;2007;35.14;1697839665
116413;1099;2007;89.19;1697845532
116415;1033;2006;45.24;1697851510
116418;1038;2002;68.75;1697853629
116422;1066;2006;66.67;1697859440
116427;1093;2004;19.05;1697861909
116433;1075;2003;14.81;1697863650
116440;1063;2010;77.27;1697863658
116448;1081;2009;73.68;1697867499
116457;1044;2005;47.62;1697870169
116467;1091;2010;100.00;1697873558
116478;1086;2005;85.71;1697878383
116490;1064;2006;47.62;1697880150
116503;1050;2001;21.74;1697884918
116517;1044;2008;50.00;1697886299
116532;1035;2006;71.43;1697891733
116548;1060;2003;100.00;1697894737
116565;1001;2010;4.55;1697897913
116583;1099;2003;96.30;1697899240
116602;1093;2005;4.76;1697899497
116622;1023;2005;47.62;1697902287
116643;1059;2003;100.00;1697904965
116665;1081;2007;2.70;1697910302
116688;1072;2010;100.00;1697915965
116712;1071;2007;27.03;1697917106
116737;1033;2005;66.67;1697919785
116763;1090;2002;100.00;1697924064
116790;1040;2005;14.29;1697927944
116818;1011;2005;16.67;1697929401
116847;1041;2002;100.00;1697931333
116877;1066;2010;0.00;1697934779
116908;1051;2006;14.29;1697935230
116940;1046;2002;84.38;1697935769
116973;1056;2007;32.43;1697940970
117007;1060;2008;15.62;1697945748
117042;1051;2010;100.00;1697949870
117078;1070;2004;21.43;1697953633
117115;1068;2007;100.00;1697954103
117153;1057;2009;18.42;1697954194
117192;1021;2005;92.86;1697958390
117232;1011;2010;22.73;1697961402
117273;1016;2010;100.00;1697966717
117315;1070;2010;59.09;1697967578
117358;1055;2007;100.00;1697971127
117402;1061;2009;81.58;1697974837
117447;1031;2003;100.00;1697978291
117493;1068;2010;100.00;1697980739
117540;1067;2009;21.05;1697983246
117588;1049;2010;86.36;1697984817
117637;1004;2003;25.93;1697985330
117687;1094;2007;40.54;1697986705
117738;1046;2010;100.00;1697989748
117790;1060;2002;46.88;1697994510
117843;1089;2001;100.00;1697997595
117897;1074;2003;25.93;1698002525
117952;1003;2004;100.00;1698003029
118008;1098;2001;100.00;1698004994
118065;1019;2005;83.33;1698008969
118123;1008;2002;40.62;1698013486
118182;1001;2002;100.00;1698014733
118242;1005;2001;69.57;1698015232
118303;1040;2007;2.70;1698016322
118365;1029;2002;21.88;1698020278
118428;1084;2001;100.00;1698021499
118492;1080;2006;66.67;1698022069
118557;1025;2010;40.91;1698025437
118623;1041;2002;9.38;1698028431
118690;1072;2005;100.00;1698031528
118758;1080;2005;90.48;1698035451
118827;1005;2009;68.42;1698038348
118897;1098;2006;23.81;1698043916
118968;1052;2006;100.00;1698045203
119040;1015;2004;80.95;1698049473
119113;1052;2003;0.00;1698053421
119187;1003;2010;100.00;1698059189
119262;1004;2006;61.90;1698064567
119338;1088;2009;71.05;1698070169
119415;1020;2002;53.12;1698074780
119493;1014;2006;100.00;1698078469
119572;1024;2003;100.00;1698081579
119652;1012;2006;42.86;1698085297
119733;1007;2002;6.25;1698091225
119815;1086;2003;29.63;1698092760
119898;1007;2007;86.49;1698094397
119982;1023;2003;100.00;1698099727
120067;1047;2008;71.88;1698104525
120153;1054;2006;4.76;1698107546
120240;1025;2002;31.25;1698108023
120328;1012;2006;28.57;1698110870
120417;1001;2001;39.13;1698116185
120507;1036;2008;46.88;1698120572
120598;1082;2002;84.38;1698122200
120690;1073;2001;100.00;1698123059
120783;1007;2003;85.19;1698123256
120877;1098;2003;51.85;1698126528
120972;1016;2003;100.00;1698130896
121068;1024;2005;100.00;1698136408
121165;1002;2002;6.25;1698137336
121263;1025;2010;63.64;1698139692
121362;1041;2007;100.00;1698143268
121462;1077;2006;73.81;1698148967
121563;1029;2001;65.22;1698151675
121665;1085;2010;100.00;1698155163
121768;1079;2007;100.00;1698157146
121872;1093;2004;100.00;1698161377
121977;1056;2009;7.89;1698165675
122083;1022;2006;100.00;1698169798
122190;1041;2007;91.89;1698175489
122298;1041;2009;5.26;1698179452
122407;1013;2002;6.25;1698181429
122517;1006;2005;2.38;1698181743
122628;1021;2003;44.44;1698186723
122740;1036;2007;48.65;1698187427
122853;1099;2007;100.00;1698191198
122967;1065;2008;93.75;1698194001
123082;1040;2007;83.78;1698197032
123198;1037;2002;81.25;1698200927
123315;1098;2004;100.00;1698201763
123433;1092;2001;100.00;1698203401
123552;1044;2009;7.89;1698207034
123672;1047;2006;0.00;1698210156
123793;1032;2008;100.00;1698211348
123915;1070;2004;88.10;1698213786
124038;1099;2006;76.19;1698219149
124162;1095;2003;14.81;1698224856
124287;1075;2010;13.64;1698227176
124413;1018;2002;90.62;1698228448
124540;1049;2006;78.57;1698233918
124668;1058;2006;100.00;1698239707
124797;1092;2004;100.00;1698245111
124927;1022;2005;88.10;1698245112
125058;1002;2008;100.00;1698249971
125190;1044;2001;100.00;1698255346
125323;1017;2004;83.33;1698258823
125457;1094;2007;75.68;1698261411
125592;1033;2007;100.00;1698266683
125728;1086;2008;84.38;1698270782
125865;1033;2003;37.04;1698274417
126003;1047;2009;34.21;1698277135
126142;1027;2002;0.00;1698278447
126282;1024;2003;100.00;1698283621
126423;1047;2004;0.00;1698283844
126565;1065;2006;7.14;1698287336
126708;1058;2005;100.00;1698288263
126852;1096;2006;100.00;1698289081
126997;1030;2008;68.75;1698294973
127143;1082;2006;52.38;1698295999
127290;1082;2003;100.00;1698296480
127438;1025;2001;100.00;1698297356
127587;1004;2001;95.65;1698299327
127737;1068;2010;100.00;1698305313
127888;1083;2007;40.54;1698307672
128040;1061;2003;66.67;1698309836
128193;1046;2007;45.95;1698311724
128347;1014;2005;88.10;1698312762
128502;1053;2003;51.85;1698317174
128658;1088;2005;19.05;1698322334
128815;1090;2005;19.05;1698326465
128973;1080;2001;21.74;1698329617
129132;1038;2005;100.00;1698333021
129292;1006;2004;21.43;1698333878
129453;1014;2003;37.04;1698337660
129615;1066;2002;93.75;1698343450
129778;1096;2009;100.00;1698347526
129942;1045;2008;100.00;1698348370
130107;1070;2006;100.00;1698353032
130273;1037;2005;100.00;1698356147
130440;1093;2002;100.00;1698357311
130608;1057;2004;100.00;1698357695
130777;1097;2001;95.65;1698363534
130947;1035;2003;85.19;1698366375
131118;1017;2002;25.00;1698370189
131290;1062;2010;100.00;1698375099
131463;1089;2008;15.62;1698379768
131637;1042;2008;3.12;1698383576
131812;1007;2005;64.29;1698383788
131988;1069;2008;81.25;1698384650
132165;1034;2009;2.63;1698390377
132343;1073;2001;95.65;1698395336
132522;1060;2003;62.96;1698399320
132702;1079;2003;40.74;1698403729
132883;1091;2005;33.33;1698408613
133065;1054;2009;92.11;1698408949
133248;1066;2008;87.50;1698410164
133432;1001;2005;11.90;1698413865
133617;1097;2002;31.25;1698416391
133803;1033;2008;93.75;1698420276
133990;1002;2009;0.00;1698424476
134178;1030;2010;100.00;1698427531
134367;1002;2001;26.09;1698433320
134557;1032;2009;100.00;1698439212
134748;1088;2005;100.00;1698439765
134940;1017;2005;23.81;1698444036
135133;1004;2009;10.53;1698446839
135327;1073;2009;73.68;1698448553
135522;1074;2004;78.57;1698451490
135718;1020;2006;73.81;1698454923
135915;1051;2007;27.03;1698458231
136113;1055;2004;59.52;1698463573
136312;1068;2006;100.00;1698466329
136512;1042;2001;52.17;1698467555
136713;1066;2001;86.96;1698471186
136915;1037;2005;30.95;1698475853
137118;1043;2003;74.07;1698480912
137322;1058;2003;44.44;1698486008
137527;1007;2009;100.00;1698490231
137733;1009;2009;84.21;1698493219
137940;1037;2009;76.32;1698496921
138148;1061;2010;95.45;1698497736
138357;1018;2001;21.74;1698500468
138567;1082;2006;92.86;1698502213
138778;1086;2005;30.95;1698504885
138990;1028;2006;42.86;1698510186
139203;1086;2003;25.93;1698510589
139417;1023;2001;91.30;1698513135
139632;1094;2009;78.95;1698513618
139848;1023;2002;65.62;1698516212
140065;1057;2007;70.27;1698522077
140283;1058;2001;100.00;1698526300
140502;1037;2006;40.48;1698529604
140722;1043;2004;11.90;1698532038
140943;1027;2006;57.14;1698532291
141165;1082;2002;100.00;1698533078
141388;1085;2001;100.00;1698537251
141612;1045;2003;7.41;1698537561
141837;1010;2010;86.36;1698537987
142063;1033;2010;63.64;1698538404
142290;1044;2005;100.00;1698538990
142518;1082;2010;100.00;1698541572
142747;1001;2003;100.00;1698546776
142977;1095;2002;6.25;1698547693
143208;1060;2007;97.30;1698548506
143440;1085;2010;100.00;1698552931
143673;1040;2009;100.00;1698553400
143907;1005;2006;52.38;1698553748
144142;1013;2007;21.62;1698555542
144378;1028;2007;40.54;1698557319
144615;1089;2003;74.07;1698563232
144853;1046;2003;55.56;1698566175
145092;1090;2004;76.19;1698566623
145332;1007;2008;78.12;1698570925
145573;1007;2005;2.38;1698576858
145815;1050;2010;27.27;1698580620
146058;1048;2004;2.38;1698583286
146302;1074;2004;57.14;1698583374
146547;1058;2007;40.54;1698587411
146793;1018;2007;0.00;1698589099
147040;1010;2002;100.00;1698593266
147288;1059;2009;81.58;1698594129
147537;1016;2009;84.21;1698597076
147787;1041;2007;100.00;1698601234
148038;1059;2003;44.44;1698602611
148290;1055;2006;21.43;1698606516
148543;1091;2002;59.38;1698606853
148797;1019;2008;81.25;1698609585
149052;1023;2008;25.00;1698612984
149308;1024;2005;47.62;1698614369
149565;1037;2002;50.00;1698618972
149823;1004;2004;100.00;1698621231
150082;1074;2003;18.52;1698622200
150342;1098;2004;95.24;1698626872
150603;1074;2002;40.62;1698631335
150865;1041;2006;76.19;1698634471
151128;1012;2003;33.33;1698639048
151392;1091;2003;3.70;1698642485
151657;1069;2002;75.00;1698643001
151923;1063;2003;55.56;1698647819
152190;1099;2006;30.95;1698648203
152458;1036;2003;51.85;1698651463
152727;1048;2010;45.45;1698652271
152997;1034;2005;38.10;1698656608
153268;1040;2005;0.00;1698657264
153540;1059;2004;100.00;1698662860
153813;1015;2009;52.63;1698668104
154087;1064;2001;100.00;1698673145
154362;1075;2002;100.00;1698677037
154638;1007;2009;50.00;1698677686
154915;1057;2008;81.25;1698681231
155193;1027;2009;34.21;1698685080
155472;1070;2008;40.62;1698688454
155752;1098;2006;92.86;1698689390
156033;1012;2009;34.21;1698690548
156315;1071;2004;38.10;1698693497
156598;1034;2009;5.26;1698698824
156882;1044;2001;82.61;1698704194
157167;1072;2002;100.00;1698707078
157453;1061;2006;66.67;1698710481
157740;1005;2001;47.83;1698716389
158028;1046;2006;92.86;1698722328
158317;1091;2007;72.97;1698724375
158607;1025;2005;100.00;1698725286
158898;1041;2003;29.63;1698727002
159190;1076;2010;100.00;1698729125
159483;1072;2002;100.00;1698735096
159777;1073;2006;97.62;1698740808
160072;1080;2006;0.00;1698745967
160368;1042;2005;30.95;1698746281
160665;1035;2005;78.57;1698750359
160963;1068;2003;14.81;1698754588
161262;1018;2010;9.09;1698758793
161562;1083;2008;81.25;1698762139
161863;1051;2006;85.71;1698762995
162165;1012;2006;90.48;1698766690
162468;1003;2002;100.00;1698768668
162772;1061;2008;100.00;1698771857
163077;1048;2002;59.38;1698773784
163383;1024;2007;16.22;1698774562
163690;1085;2010;13.64;1698779763
163998;1079;2003;29.63;1698780174
164307;1025;2007;54.05;1698781495
164617;1095;2004;40.48;1698784164
164928;1046;2004;59.52;1698785198
165240;1055;2006;52.38;1698790217
165553;1052;2005;38.10;1698791785
165867;1023;2008;46.88;1698793932
166182;1012;2010;13.64;1698797929
166498;1048;2008;31.25;1698803851
166815;1021;2007;89.19;1698804344
167133;1074;2001;21.74;1698805524
167452;1084;2001;0.00;1698810814
167772;1089;2006;100.00;1698816598
168093;1050;2005;16.67;1698820877
168415;1085;2001;60.87;1698821794
168738;1080;2001;100.00;1698827641
169062;1088;2007;24.32;1698832337
169387;1026;2004;64.29;1698832437
169713;1024;2007;13.51;1698838081
170040;1100;2001;100.00;1698838794
170368;1013;2006;100.00;1698842637
170697;1085;2008;90.62;1698847850
171027;1003;2009;28.95;1698847974
171358;1069;2001;30.43;1698853523
171690;1052;2002;100.00;1698859100
172023;1046;2007;100.00;1698864976
172357;1084;2008;31.25;1698868094
172692;1001;2004;83.33;1698873559
173028;1098;2009;39.47;1698878070
173365;1097;2005;38.10;1698882728
173703;1011;2003;100.00;1698886565
174042;1040;2007;8.11;1698887947
174382;1084;2006;97.62;1698890136
174723;1010;2004;95.24;1698890158
175065;1100;2002;90.62;1698891039
175408;1063;2008;100.00;1698897039
175752;1038;2006;42.86;1698899482
176097;1044;2001;100.00;1698902006
176443;1041;2002;15.62;1698903825
176790;1095;2006;100.00;1698907584
177138;1070;2010;13.64;1698910784
177487;1010;2009;18.42;1698916586
177837;1002;2010;50.00;1698917534
178188;1034;2008;93.75;1698917890
178540;1035;2005;100.00;1698921556
178893;1086;2004;35.71;1698922494
179247;1074;2006;66.67;1698924118
179602;1063;2001;56.52;1698924472
179958;1011;2006;59.52;1698925176
180315;1032;2004;54.76;1698927630
180673;1057;2004;54.76;1698932873
181032;1025;2010;45.45;1698936449
181392;1005;2004;28.57;1698939382
181753;1076;2002;18.75;1698940434
182115;1088;2006;0.00;1698945017
182478;1032;2002;100.00;1698947872
182842;1091;2010;100.00;1698950776
183207;1049;2005;100.00;1698954941
183573;1011;2001;26.09;1698958848
183940;1056;2010;90.91;1698961490
