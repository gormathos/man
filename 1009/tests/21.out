94
21.10.2023 07:32:38
21.10.2023 13:25:37
21.10.2023 16:16:05
21.10.2023 19:19:25
21.10.2023 21:34:24
21.10.2023 23:35:33
22.10.2023 04:44:30
22.10.2023 05:55:03
22.10.2023 09:25:17
22.10.2023 10:38:47
22.10.2023 12:38:11
22.10.2023 13:18:59
22.10.2023 15:49:08
22.10.2023 17:59:55
22.10.2023 19:30:29
22.10.2023 20:03:14
22.10.2023 22:45:33
23.10.2023 00:38:19
23.10.2023 03:25:28
23.10.2023 07:13:23
23.10.2023 11:06:29
23.10.2023 16:27:49
23.10.2023 17:19:39
23.10.2023 22:22:07
24.10.2023 04:50:59
24.10.2023 07:01:36
24.10.2023 08:33:28
24.10.2023 10:27:48
24.10.2023 13:46:03
24.10.2023 14:19:06
24.10.2023 15:29:37
24.10.2023 17:49:58
24.10.2023 23:46:38
25.10.2023 02:42:43
25.10.2023 03:10:01
25.10.2023 05:22:28
25.10.2023 13:15:07
25.10.2023 14:45:11
25.10.2023 16:06:11
25.10.2023 17:35:46
25.10.2023 20:44:43
26.10.2023 01:27:01
26.10.2023 02:44:23
26.10.2023 02:58:01
26.10.2023 05:01:20
26.10.2023 05:15:56
26.10.2023 07:28:33
26.10.2023 15:10:21
26.10.2023 19:12:06
26.10.2023 19:26:10
26.10.2023 20:43:52
26.10.2023 21:35:47
26.10.2023 21:55:11
26.10.2023 22:01:35
27.10.2023 02:51:39
27.10.2023 17:25:31
27.10.2023 20:40:12
27.10.2023 20:49:25
28.10.2023 04:12:09
28.10.2023 10:50:31
28.10.2023 20:51:40
28.10.2023 22:44:38
28.10.2023 23:54:11
29.10.2023 00:23:10
29.10.2023 01:06:12
29.10.2023 02:32:56
29.10.2023 04:15:31
29.10.2023 04:23:20
29.10.2023 15:27:46
29.10.2023 17:40:34
29.10.2023 23:13:51
30.10.2023 10:47:40
30.10.2023 13:39:05
30.10.2023 14:43:57
30.10.2023 23:04:38
31.10.2023 04:08:06
31.10.2023 05:12:05
31.10.2023 06:51:36
31.10.2023 16:11:08
31.10.2023 17:04:17
01.11.2023 05:29:58
01.11.2023 08:34:01
01.11.2023 11:39:54
01.11.2023 12:43:57
01.11.2023 17:18:20
01.11.2023 18:56:16
02.11.2023 00:56:05
02.11.2023 03:50:39
02.11.2023 05:13:26
02.11.2023 06:46:24
02.11.2023 10:39:16
02.11.2023 17:57:52
02.11.2023 18:46:16
02.11.2023 19:55:41
